dotcom-rendering/src/components/MostViewedFooterData.importable.tsx

```diff
@@ -52,20 +51,23 @@ interface MostViewedFooterPayloadType {
 
 export const MostViewedFooterData = ({
 	sectionId,
-	format,
 	ajaxUrl,
 	edition,
 }: Props) => {
 	// Example usage of AB Tests
 	// Used in the Cypress tests as smoke test of the AB tests framework integration
 	const ABTestAPI = useAB()?.api;
 
-	const abTestCypressDataAttr =
-		(ABTestAPI?.isUserInVariant('AbTestTest', 'control') &&
-			'ab-test-control') ||
-		(ABTestAPI?.isUserInVariant('AbTestTest', 'variant') &&
-			'ab-test-variant') ||
-		'ab-test-not-in-test';
+	let abTestCypressDataAttr = 'ab-test-not-in-test';
+
+	if (ABTestAPI?.isUserInVariant('AbTestTest', 'control')) {
+		abTestCypressDataAttr = 'ab-test-control';
+	}
+
+	if (ABTestAPI?.isUserInVariant('AbTestTest', 'variant')) {
+		abTestCypressDataAttr = 'ab-test-variant';
+	}
+
```

Comment:
We could also keep a nested ternary if we wanted to avoid mutation here.

Reply:
yeah i went with this because it took me several minutes to work out this was logic here - i hoped this was more clear, and worth the local mutation?